adafruit_esp32spi/adafruit_esp32spi.py

```diff
@@ -527,11 +527,15 @@ def socket_write(self, socket_num, buffer):
         if self._debug:
             print("Writing:", buffer)
         self._socknum_ll[0][0] = socket_num
-        resp = self._send_command_get_response(_SEND_DATA_TCP_CMD,
-                                               (self._socknum_ll[0], buffer),
-                                               sent_param_len_16=True)
+        chunk_size = 64
+        sent = 0
+        for chunk in range((len(buffer) // chunk_size)+1):
+            chunk_buffer = buffer[(chunk*chunk_size):((chunk+1)*chunk_size)]
+            resp = self._send_command_get_response(_SEND_DATA_TCP_CMD,
```

Comment:
won't this alway send multiples of  64 bytes - even in len(buffer) is not a multiple of 64?
(if len(buffer) < 64 will it send 64?)
If so won't that fail the test at line 539?

Reply:
thanks for the comment, @jerryneedell ! i was deep into trying to find where the Python docs answer this question before I saw your edit, so I'm pasting this here for posterity:

https://docs.python.org/3/library/stdtypes.html#common-sequence-operations

I find the docs pretty unreadable but my basic understanding is that the end of a slice is a max value, and Python will stop extracting bytes when the buffer being sliced runs out, or when max value is reached.

I was able to confirm by playing on the REPL:

```python
>>> (b'foo bar baz')[0:100]
b'foo bar baz'
```